src/builder/modules/Generator_Builder.res

```diff
@@ -457,8 +490,13 @@ let getOffset = ((x1, y1): position, (x2, y2): position) => {
   resolution: https://physics.info/vector-components/summary.shtml This results
   in a fully opaque line with the correct width if the line is vertical or
   horizontal, but antialiasing may still affect lines at other angles.
+
+  The angle should be 3 pi / 2 radians if the angle is 270 degrees, but with 
+  the ternary making it pi / 2 regardless of sign the rectangle lines functions
+  are much easier to do, for some reason. Maybe later, this could be replaced 
+  with something using the absolute value of the angle?
  */
-  let angle = Js.Math.atan2(~y=h, ~x=w, ())
+  let angle = w === 0.0 ? Js.Math._PI /. 2.0 : Js.Math.atan2(~y=h, ~x=w, ())
```

Comment:
It would be good to think about this one some more.

In the current code, you can control if a line is offset to the "left" or offset to the "right" by just changing the direction the line is drawn, e.g. instead of A to B, draw as B to A.

With this change using `w === 0`, it means that vertical lines are always offset to one side only independent of the point order, however all other lines continue to have the previous offset behaviour (adjusting depending on the order of the points), so the `w ===0` is inconsistent with all lines where `w !== 0`.

For now I'm thinking we leaving this as-is and change any calling code to adjust it's point drawing order to get the desired result.